clients/rust/src/generated/instructions/burn_v2.rs

```diff
@@ -316,9 +315,9 @@ impl BurnV2Builder {
             leaf_delegate: self.leaf_delegate,
             merkle_tree: self.merkle_tree.expect("merkle_tree is not set"),
             core_collection: self.core_collection,
-            mpl_core_cpi_signer: self.mpl_core_cpi_signer.unwrap_or(solana_program::pubkey!(
-                "CbNY3JiXdXNE9tPNEk1aRZVEkWdj2v7kfJLNQwZZgpXk"
-            )),
+            mpl_core_cpi_signer: self
+                .mpl_core_cpi_signer
+                .expect("mpl_core_cpi_signer is not set"),
```

Comment:
I guess we just lose the default behavior in the Rust client, that's probably OK

Reply:
I vaguely remember seeing something like this but I think it's worth just punting for now.